src/main/java/harmonised/pmmo/events/impl/EnchantHandler.java

```diff
@@ -25,17 +25,15 @@ public static void handle(EnchantEvent event) {
 			CompoundTag dataIn = TagBuilder.start()
 					.withString(APIUtils.STACK, event.getItem().serializeNBT().getAsString())
 					.withString(APIUtils.PLAYER_ID, event.getEntity().getUUID().toString())
-					.withInt(APIUtils.ENCHANT_LEVEL, event.getEnchantment().level)
-					.withString(APIUtils.ENCHANT_NAME, event.getEnchantment().enchantment.getDescriptionId()).build();
+					.withInt(APIUtils.ENCHANT_LEVELS_SPENT, event.getLevelsSpent()).build();
 			hookOutput = core.getEventTriggerRegistry().executeEventListeners(EventType.ENCHANT, event, dataIn);
 		}
 		hookOutput = TagUtils.mergeTags(hookOutput, core.getPerkRegistry().executePerk(EventType.ENCHANT, event.getEntity(), hookOutput));
 		if (serverSide) {
-			double proportion = (double)event.getEnchantment().level / (double)event.getEnchantment().enchantment.getMaxLevel();
```

Comment:
This fundamentally breaks how enchantment works in pmmo in a way that would break the configurations of other users.  The design of enchantment experience is such that the configured XP to be awarded is given proportionate to the maximum level of the enchantment.  Therefore Mending 1 gives full xp and Smite 1 gives 0.2x the xp.